content/docs/command-reference/remote/modify.md

```diff
@@ -890,7 +901,7 @@ by HDFS. Read more about by expanding the WebHDFS section in
   kerberos must be enabled for this to work.
 
   ```dvc
-  $ dvc remote modify myremote proxy_to some_proxy_user
+  $ dvc remote modify --local myremote proxy_to some_proxy_user
```

Comment:
...and here

Reply:
and here - all look like per project configs

Reply:
Reverted.